automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/notebooks/Notebook.scala

```diff
@@ -26,7 +26,7 @@ object Notebook extends RestClient with LazyLogging {
     mapper.readValue(response, classOf[NotebookContentItem])
 
   def notebooksBasePath(googleProject: GoogleProject, clusterName: ClusterName): String =
-    s"notebooks/${googleProject.value}/${clusterName.string}"
+    s"proxy/${googleProject.value}/${clusterName.string}/jupyter"
```

Comment:
does this change how ui access notebook?

Reply:
UI is already using `/proxy` for everything as we recommend.. this is just making tests use the correct path.